rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -482,7 +482,7 @@ def _get_staking_events(
             all_new_events.extend(new_events)
 
         # Request existing DB addresses' events
-        if existing_addresses and min_from_timestamp <= to_timestamp:
+        if existing_addresses and to_timestamp > min_from_timestamp:
```

Comment:
so here the problem was the `=` case in which you also queried when you should not?

Reply:
Yep, I hope twisting them gives more readability.